core/src/main/java/org/neo4j/graphalgo/api/CypherGraphSetup.java

```diff
@@ -0,0 +1,131 @@
+package org.neo4j.graphalgo.api;
+
+import org.neo4j.kernel.internal.Version;
+
+import java.util.concurrent.ExecutorService;
+
+/**
+ * @author mh
+ * @since 27.04.17
+ */
+public class CypherGraphSetup implements GraphDefinition {
```

Comment:
For example, tests like [GraphNegativeTest](https://github.com/neo4j-contrib/neo4j-graph-algorithms/blob/32ff4065dbb324c5965d44718a8e111f2e716086/tests/src/test/java/org/neo4j/graphalgo/core/GraphNegativeTest.java#L36-L43), [PageRankTest](https://github.com/neo4j-contrib/neo4j-graph-algorithms/blob/32ff4065dbb324c5965d44718a8e111f2e716086/tests/src/test/java/org/neo4j/graphalgo/impl/PageRankTest.java#L32-L39), or [ShortestPathDijkstraTest](https://github.com/neo4j-contrib/neo4j-graph-algorithms/blob/32ff4065dbb324c5965d44718a8e111f2e716086/tests/src/test/java/org/neo4j/graphalgo/impl/ShortestPathDijkstraTest.java#L29-L36) run via junit parameters on all graph impls and we could include cypher loading there as well.